code401challenges/src/main/java/code401challenges/LinkedList.java

```diff
@@ -51,5 +51,51 @@ public String toString(){
         }
         return list.toString();
     }
+
+    public Node append(int key){
+        Node newNode = new Node();
+        newNode.value = key;
+        Node current = head;
+        while(current != null){
+            if(current.next == null){
+                current.next = newNode;
+                break;
+            }
+            current = current.next;
+        }
+        return newNode;
+    }
```

Comment:
Looks good! You could also just use return in the loop, instead of using break, but that's your choice.